.github/workflows/release.yml

```diff
@@ -1,33 +1,85 @@
+        id: release
+        with:
+          release-type: node
+          package-name: puppeteer
+          command: github-release
+
+  publish:
+    needs: release-pr
+    if: ${{ needs.release-pr.outputs.release_created }}
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Install dependencies
+        run: npm install
+      - name: Build puppeteer
+        run: npm run build
+      - name: Publish puppeteer
+        env:
+          NPM_TOKEN: ${{secrets.NPM_TOKEN_PUPPETEER}}
+        run: |
+          npm config set registry 'https://wombat-dressing-room.appspot.com/'
+          npm config set '//wombat-dressing-room.appspot.com/:_authToken' '${NPM_TOKEN}'
+          echo "Publishing puppeteer"
+          npm publish
+          # DEPRECATED_RANGE=$(node utils/get_deprecated_version_range.js)
+          # echo "Deprecating old puppeteer versions: $DEPRECATED_RANGE"
+          # npm deprecate puppeteer@$DEPRECATED_RANGE "Version no longer supported. Upgrade to @latest"
+      - name: Publish puppeteer-core
+        env:
+          NPM_TOKEN: ${{secrets.NPM_TOKEN_PUPPETEER_CORE}}
+        run: |
+          utils/prepare_puppeteer_core.js
+          npm config set registry 'https://wombat-dressing-room.appspot.com/'
+          npm config set '//wombat-dressing-room.appspot.com/:_authToken' '${NPM_TOKEN}'
+          npm publish
+
+  post-publish:
+    needs: publish
     runs-on: ubuntu-latest
     permissions:
       contents: write
-      pull-requests: write
     steps:
       - name: Checkout
         uses: actions/checkout@v3
-      - name: Configure git
+      - name: Append `-post` to version
+        id: post-version
         run: |
-          git config --global user.name 'Puppeteer Bot'
-          git config --global user.email '<puppeteer@chromium.org>'
-      - name: Release
-        id: release
+          npm version --no-git-tag-version --no-commit-hooks "$(jq -r .version ./package.json)-post"
+          echo ::set-output name=VERSION::$(jq -r .version ./package.json)
+      - name: Build
         run: |
           npm install
-          npm run release
-          echo ::set-output name=VERSION::$(jq -r .version ./package.json)
-      - name: Create pull request
-        uses: peter-evans/create-pull-request@v4
-        with:
-          branch: releases/v${{ steps.release.outputs.VERSION }}
-          delete-branch: true
-          commit-message: 'chore(release): mark v${{ steps.release.outputs.VERSION }}'
-          title: 'chore(release): mark v${{ steps.release.outputs.VERSION }}'
-          body: ''
-          labels: release
-          reviewers: jrandolf, OrKoN, ergunsh
+          npm run build
+          npm run doc
+      - name: Configure git
+        run: |
+          git config --global user.name 'actions-bot'
+          git config --global user.email '<github-actions-bot@google.com>'
```

Comment:
is it a new group email address?

Reply:
No, this is what the action uses for their own commit:
- https://github.com/google-github-actions/release-please-action/blob/main/.github/workflows/release-please.yaml
It's safe to use for commits.